server/implementation/src/main/java/io/smallrye/graphql/execution/error/ExceptionHandler.java

```diff
@@ -19,15 +24,21 @@
 
     private final Config config;
     private final ExceptionLists exceptionLists;
+    protected List<String> unwrapExceptions = new ArrayList<>();
 
     public ExceptionHandler(Config config) {
         this.config = config;
         this.exceptionLists = new ExceptionLists(config.getHideErrorMessageList(), config.getShowErrorMessageList());
+        if (config.getUnwrapExceptions().isPresent()) {
+            this.unwrapExceptions.addAll(config.getUnwrapExceptions().get());
+        }
+        this.unwrapExceptions.addAll(DEFAULT_EXCEPTION_UNWRAP);
```

Comment:
Except for the `DataFetcherException`, this (and the constant) is a code duplication with `AbstractDataFetcher`. Maybe it would be better to push that into the `Config` class, so this could would be only `this.unwrapExceptions = config.getAllUnwrapExceptions()`.

Reply:
It's been removed (or rather moved) from `AbstractDataFetcher.` So it's not in `AbstractDataFetcher` anymore